client/my-sites/stats/hooks/use-stats-purchases.tsx

```diff
@@ -89,27 +88,18 @@ export default function useStatsPurchases( siteId: number | null ) {
 		return isProductOwned( sitePurchases, PRODUCT_JETPACK_STATS_FREE );
 	}, [ sitePurchases ] );
 
-	const isCommercialOwned = useMemo( () => {
-		return areProductsOwned( sitePurchases, [
-			// TODO: Remove the VideoPress products after releasing the feature flag `stats/restricted-dashboard`.
-			...JETPACK_VIDEOPRESS_PRODUCTS,
```

Comment:
If we are removing the VideoPress checks here, then I guess we'll need to remove  Video stats from paywall?

https://github.com/Automattic/wp-calypso/blob/beee63d387a8fa7f4b5f6e0255e7ce0c76d0901c/client/my-sites/stats/hooks/use-should-gate-stats.ts#L53

Reply:
Did we **_temporarily_** allow the VideoPress product to [pass the force purchase redirection](https://github.com/Automattic/wp-calypso/pull/90838) since users who have the VideoPress purchase would not be able to visit stats? That means we would still need to paywall commercial sites from the Videos module [unless they have a VideoPress purchase](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/stats/hooks/use-should-gate-stats.ts#L127-L133), isn't it?

Reply:
Good questions!

> Did we temporarily allow the VideoPress product to https://github.com/Automattic/wp-calypso/pull/90838 since users who have the VideoPress purchase would not be able to visit stats?

Yes - to allow VP users to see the stats of the videos.

> That means we would still need to paywall commercial sites from the Videos module [unless they have a VideoPress purchase](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/stats/hooks/use-should-gate-stats.ts#L127-L133), isn't it? 🤔

Now that VP users have access to Jetpack Stats (restricted or not) and modules are gated individually, we have the ability to show them the module without any special treatment. The video module is only meant for VideoPress, and VideoPress does have a free tier. So I think we could just make it free for access in this case.

Discussion: p1HpG7-sn0-p2

cc @grbicsanja @ajaykj

Reply:
Just talked with @dognose24, the access control for VP is here: 

https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/stats/hooks/use-should-gate-stats.ts#L127-L133

Where we only allow paid plan of VP users to access the module, which I think is a better idea! Some great thinking and measures taken ahead @dognose24! Thanks!